### PR TITLE
KAN-79 - Multi Language Support: German & English

### PR DESCRIPTION
**Pull request by levy002**
#### PR description 📝
- implement multi-language support
- integrate German and English languages in all files

#### Ticket / Jira Link 🔗
- [KAN-79](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=63726b3c3c26ca7fa0d07665&selectedIssue=KAN-79)

#### Checklist ✅
- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings 

#### How To Test Changes
- Clone this repository

``` sh 
  git git@github.com:levy002/certificate-manager--Levy.git
``` 

- checkout to this branch `ft/task7-KAN-79-Multi-language-Support`

- Open the app in your browser by running this command in your terminal

```
  npm install
  npm start
```

##### Thanks for reviewing my PR 😎

### CODE REVIEW

File: src/components/certificates/EditCertificate.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const EditCertificate: React.FC = () => {
   }
 
   if (!certificate) {
-    return <p>No certificate found with id: {id}</p>;
+    return <p>No certificate found</p>;
```
Comment (jtrbalic) [inline src/components/certificates/EditCertificate.tsx]: translate this as well

File: src/locales/de.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "Start": "Starten",
```
Comment (jtrbalic) [inline src/locales/de.json]: Only thing here is that keys should be really keys not translation. So here you have key:value pair. So keys should be named as variables using english (no spaces, upercases but camel_case)

"start": "Starten",
"machine_learning": "Maschinelles Lernen",
"page_not_founr": "Entschuldigung, die angeforderte Seite konnte nicht gefunden werden. Versuchen Sie, zurück nach Hause zu navigieren.",

File: src/components/certificates/CertificateForm.tsx [hunk 3]
```diff
@@ -156,7 +162,7 @@ const CertificateForm: React.FC<CertificateFormProps> = ({
             </div>
 
             <SelectField
-              label="Certificate type"
+              label="Certificate Type"
```
Comment (AidaLepicDccs) [inline src/components/certificates/CertificateForm.tsx]: Really small, but this should also be translated. Any text gets translated - descriptive text, labels, headers, placeholders, etc.

File: src/components/certificates/CertificateTable.tsx [hunk 4]
```diff
@@ -57,10 +59,12 @@ const CertificatesTable: React.FC<CertificateTableProps> = ({
             <thead className="table__head">
               <tr className="table__row">
                 <th className="table__header" />
-                <th className="table__header">Supplier</th>
-                <th className="table__header">Certificate Type</th>
-                <th className="table__header">Valid from</th>
-                <th className="table__header">Valid to</th>
+                <th className="table__header">{translate('supplier')}</th>
+                <th className="table__header">
+                  {translate('Certificate Type')}
```
Comment (AidaLepicDccs) [inline src/components/certificates/CertificateTable.tsx]: This should be in key form: Certificate Type -> certificate_type.

**Review by AidaLepicDccs — APPROVED** [review]

**Pull request merged**